src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -605,6 +605,15 @@
 	IEmoji getEmojiByName(String name);
 
 	/**
+	 * This adds a custom emoji to the guild.
+	 *
+	 * @param name The name, <b>without colons</b>.
+	 * @param image The BufferedImage of the Emoji.
+	 * @return The emoji.
```

Comment:
I dont see any @throws DiscordException anywhere else, so I left that out, but everything else I updated.